scripts/bindings.py

```diff
@@ -98,27 +102,27 @@ def generic_function(function):
     """
 
     signature = function
-    rtype, function = function.split(' ', 1)
+    rtype, function = function.split(None, 1)
     fname, fargs = function.split('(', 1)
 
     fname = fname.replace('*', '')
     fargs = fargs.replace(');', '')
 
-    text = (
-        '# - {signature}\n'
-        'def {fname}(*args):\n'
-        '    """\n'
-        '    Describe the function.\n'
-        '\n'
-        '    :param args: arguments\n'
-        '    :return: value\n'
-        '    """\n'
-        '\n'
-        '    # TODO\n'
-        '    return clibui.{fname}()\n\n\n'
-    )
+    return '''\
+# - {signature}
+def {fname}(*args):
+    """
+    Describe the function.
+
+    :param args: arguments
+    :return: value
+    """
+
+    # TODO
+    return clibui.{fname}()
+
 
-    return text.format(fname=fname, signature=signature, args=fargs)
+'''.format(fname=fname, signature=signature, args=fargs)
```

Comment:
On Tue, 07 Jun 2016 01:55:23 -0700
João Ventura <notifications@github.com> wrote:
> >  
> > -    return text.format(fname=fname, signature=signature, args=fargs)
> > +'''.format(fname=fname, signature=signature, args=fargs)
> 
> Visually complicated as hell. In this case I separated text = something
> from the return to simplify it visually.

Well just change it to the style you prefer:

    text = '''
    ...
    '''

    return text.format(...)

Or, simplify your life:

    return text.format(**locals())

and you're done.

-- 
Mark Summerfield, Qtrac Ltd.
    diffpdfc windows command line PDF comparison tool
        http://www.qtrac.eu/diffpdfc.html